models/mpas_atm/model_mod.f90

```diff
@@ -4870,7 +4939,7 @@ subroutine convert_vert_distrib_state(state_handle, ens_size, location, quantity
 ! if all the vertical localization coord values are missing, 
 ! we don't call this routine again, and return.
 if (all(zin == missing_r8)) then ! .or. on_bound) then
-   istatus(:) = 0
+   istatus(:) = 82    ! FIXME: Ha - Why should this be zero? I created 82 if all zin are missing.
```

Comment:
I think this is fixed? remove the FIXME?

Reply:
actually, this needs to be status 0.  VERTISUNDEF locations often come in with missing_r8 as the vertical.  the original code was correct.  istatus(:) = 0

Reply:
there is an early return in `convert_vert_distrib_state` if VERTISUNDEF so I am not sure what this check was intended to catch.  Removed the 82 iststaus.